playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -15,7 +15,8 @@ if [ "$1" == "true" ] || [ "$1" == "True" ]; then
 
 	# Download comprehensive theming from github to folder /edx/app/edxapp/themes/comprehensive 
```

Comment:
why we only get comprehensive theming and not the courses?

Reply:
i'm not sure i understand the question

Reply:
let's chat about this sometime this week